lntest/itest/lnd_test.go

```diff
@@ -2173,44 +2173,70 @@ func testOpenChannelAfterReorg(net *lntest.NetworkHarness, t *harnessTest) {
 	closeReorgedChannelAndAssert(ctxt, t, net, net.Alice, chanPoint, false)
 }
 
-// testDisconnectingTargetPeer performs a test which
-// disconnects Alice-peer from Bob-peer and then re-connects them again
+// testDisconnectingTargetPeer performs a test which disconnects Alice-peer from
+// Bob-peer and then re-connects them again
 func testDisconnectingTargetPeer(net *lntest.NetworkHarness, t *harnessTest) {
 	ctxb := context.Background()
 
+	alice, err := net.NewNode("Alice", nil)
```

Comment:
I don't think we usually name other nodes Alice and Bob? Not sure if it's a problem, but could possibly be confusing.

Reply:
idt it's more confusing than have 40-50 carol and daves :P